src/main/java/com/minecolonies/colony/permissions/Permissions.java

```diff
@@ -438,9 +464,9 @@ public boolean setPlayerRank(UUID id, Rank rank, World world)
      * @param rank        Desired rank
      * @return True if succesful, otherwise false.
      */
-    private boolean addPlayer(@NotNull GameProfile gameprofile, Rank rank)
+    private boolean addPlayer(@NotNull final GameProfile gameprofile, final Rank rank)
     {
-        @NotNull Player p = new Player(gameprofile.getId(), gameprofile.getName(), rank);
+        final @NotNull Player p = new Player(gameprofile.getId(), gameprofile.getName(), rank);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AModifiersOrderCheck)